docs/production/install.md

```diff
@@ -32,8 +30,8 @@ tar -xf zulip-server-latest.tar.gz
 
 ## Step 2: Install Zulip
 
-To set up Zulip with the most common configuration, you can run the
-installer as follows:
+To set up Zulip with the most common configuration
+run the installer as follows:
```

Comment:
This language change feels like worse English; I think you want to keep the `,` but remove the `you can`.